website/mails/mails.py

```diff
@@ -95,7 +91,7 @@ def send_mail(to_addr, mail, mimetype='plain', from_addr=None, mailer=None, cele
     from_addr = from_addr or settings.FROM_EMAIL
     mailer = mailer or tasks.send_email
     subject = mail.subject(**context)
-    message = mail.text(**context) if mimetype in ('plain', 'txt') else mail.html(**context)
+    message = mail.html(**context)
```

Comment:
If we're making mail html by default why bother setting it as an agrugement in tests/test_auth.py and other files?